modules/cnf-topology-aware-lifecycle-operator-preparing-for-updates.adoc

```diff
@@ -0,0 +1,108 @@
+----
+
+. Save the image signature of the desired platform image that was mirrored. You must add the image signature to the `PolicyGenTemplate` CR for platform updates. To get the image signature, perform the following steps:
+
+.. Specify the desired {product-title} tag by running the following command:
++
+[source,terminal]
+----
+$ OCP_RELEASE_NUMBER=<release_version>
+----
+
+.. Specify the architecture of the server by running the following command:
++
+[source,terminal]
+----
+$ ARCHITECTURE=<server_architecture>
+----
+
+.. Get the release image digest from Quay by running the following command
++
+[source,terminal]
+----
+$ DIGEST="$(oc adm release info quay.io/openshift-release-dev/ocp-release:${OCP_RELEASE_NUMBER}-${ARCHITECTURE} | sed -n 's/Pull From: .*@//p')"
+----
+
+.. Set the digest algorithm by running the following command:
++
+[source,terminal]
+----
+$ DIGEST_ALGO="${DIGEST%%:*}"
+----
+
+.. Set the digest signature by running the following command:
++
+[source,terminal]
+----
+$ DIGEST_ENCODED="${DIGEST#*:}"
+----
+
+.. Get the image signature from the link:https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release/[mirror.openshift.com] website by running the following command:
++
+[source,terminal]
+----
+$ SIGNATURE_BASE64=$(curl -s "https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release/${DIGEST_ALGO}=${DIGEST_ENCODED}/signature-1" | base64 -w0 && echo)
+----
+
+.. Save the image signature to the `checksum-<OCP_RELEASE_NUMBER>.yaml` file by running the following commands:
++
+[source,terminal]
+----
+$ cat >checksum-${OCP_RELEASE_NUMBER}.yaml <<EOF
+${DIGEST_ALGO}-${DIGEST_ENCODED}: ${SIGNATURE_BASE64}
+EOF
+----
+
+. Prepare the update graph. You have two options to prepare the update graph:
+
+.. Use the OpenShift Update Service.
++
+For more information about how to set up the graph on the hub cluster, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.4/html/clusters/managing-your-clusters#deploy-the-operator-for-cincinnati[Deploy the operator for OpenShift Update Service] and link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.4/html/clusters/managing-your-clusters#build-the-graph-data-init-container[Build the graph data init container].
+
+.. Make a local copy of the upstream graph. Host the update graph on an `http` or `https` server in the disconnected environment that has access to the spoke cluster. To download the update graph, use the following command:
++
+[source,terminal]
+----
+$ curl -s https://api.openshift.com/api/upgrades_info/v1/graph?channel=stable-4.10 -o ~/upgrade-graph_stable-4.10
+----
+
+* For Operator updates, you must perform the following tasks:
++
+. Mirror the Operator catalogs. Ensure that the desired operator images are mirrored by following the procedure in the "Mirroring Operator catalogs for use with disconnected clusters" section.
```

Comment:
For a single-step procedure, use a bullet rather than number.
```suggestion
* Mirror the Operator catalogs. Ensure that the desired operator images are mirrored by following the procedure in the "Mirroring Operator catalogs for use with disconnected clusters" section.
```